backend/satellite_tools/reposync.py

```diff
@@ -1524,9 +1527,9 @@ def fix_notice(notice):
             # old suse style; we need to append the version to id
             # to get a seperate patch for every issue
             try:
-                notice['update_id'] = notice['update_id'] + '-' + notice['version']
+                notice['update_id'] = "%s-%s" % (notice['update_id'], notice['version'])
             except TypeError: # yum in RHEL5 does not have __setitem__
-                notice._md['update_id'] = notice['update_id'] + '-' + notice['version']
+                notice._md['update_id'] = "%s-%s" % (notice['update_id'], notice['version'])
```

Comment:
Looking at the comment, I'd say it is safe to remove the try/except statements.

Reply:
Do you mean because of dropping RHEL5?

Reply:
Maybe not just dropping, but in case of Uyuni as it was never build for RHEL5 :-D

Reply:
Exactly, we don't need to keep backwards compatibility for RHEL5's yum

Reply:
I just dropped one more workaround for the same issue with https://github.com/uyuni-project/uyuni/pull/4728/commits/11d3a18f30ab77b1d1db659c7ec16edefe86e5aa